server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -1152,7 +1152,21 @@ public TSStatus setStorageGroup(String storageGroup) throws TException {
   }
 
   @Override
-  public TSStatus createTimeseries(TSCreateTimeseriesReq req) {
+  public TSStatus deleteStorageGroup(TSDeleteDataReq req) {
+    if (!checkLogin()) {
+      logger.info(INFO_NOT_LOGIN, IoTDBConstant.GLOBAL_DB_NAME);
+      return new TSStatus(getStatus(TSStatusCode.NOT_LOGIN_ERROR));
+    }
+    MetadataPlan plan = new MetadataPlan(MetadataOperator.NamespaceType.DELETE_STORAGE_GROUP,new Path(req.getPaths().get(0)));
```

Comment:
So only one storage group is supported at the same time? I guess it is not difficult to support deleting multiple groups with one statement, try if you can.

Reply:
As the issue defined in IOTDB-188, I just need to delete a single storage group at the same time, and Qiaojialin said to me that before we complete the batch deletion of storage groups, we should change some of the structure of the Metadata, so he suggested me to submit this version first.

Reply:
You can always go beyond the issue as long as it is helpful, but it is fine to leave it as future work.